UnitySDK/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAcademy.cs

```diff
@@ -11,7 +11,7 @@ public override void InitializeAcademy()
         // make walker joint calculations more accurate.
         Physics.defaultSolverIterations = 12;
         Physics.defaultSolverVelocityIterations = 12;
-        Time.fixedDeltaTime = 0.01333f; //(75fps). default is .2 (60fps)
```

Comment:
These will disappear with the addition of SettingsOverride right?

Reply:
Yes.